syntax = "proto3";

package fixbug;
option cc_generic_services = true;


message ResultCode
{
    int32 errcode = 1;
    bytes errmsg = 2;
}

message GetFriendListsRequest
{
   uint32 id = 1;
}

message GetFriendListsResponse
{
    ResultCode reuslt = 1;
    repeated bytes friends = 2;
}

// 好友模块
service FriendServiceRpc
{
    rpc Register(GetFriendListsRequest) returns(GetFriendListsResponse);
}